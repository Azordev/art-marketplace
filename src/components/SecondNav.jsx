import React from "react";
import { Link } from "react-router-dom";

import Logo from "../assets/img/logo.png";

const SecondNav = ({ isLogin }) => {
	return (
		<>
			<section className="relative w-full sm:px-9 text-gray-700 bg-white body-font">
				<div className="max-w-screen-2xl container flex flex-col lg:flex-row justify-between py-5 px-0 mx-auto md:flex-row">
					<div className="flex items-center justify-center md:justify-start">
						<Link to="/">
							<img
								src={Logo}
								alt="logo"
								className="block object-contain mx-w-17-4 mx-h-4-6 lg:w-17-4"
							/>
						</Link>
					</div>
					<div className="relative flex-col lg:flex-row inline-flex items-center md:items-end lg:items-center lg:space-x-3">
						{isLogin ? (
							<>
								<span className="inline-flex items-center lg:justify-center px-14 md:px-0 lg:px-14 py-3.5 text-xl font-semibold leading-6 text-black whitespace-no-wrap">
									¿No tienes una cuenta?
								</span>
								<Link
									to="/register"
									className="inline-flex hover-scale items-center justify-center px-14 py-3.5 text-xl font-semibold leading-6 text-white whitespace-no-wrap bg-primary rounded-md shadow-button"
								>
									Regístrate aquí
								</Link>
							</>
						) : (
							<>
							<span className="inline-flex items-center lg:justify-center px-14 md:px-0 lg:px-14 py-3.5 text-xl font-semibold leading-6 text-black whitespace-no-wrap">
									¿Ya tienes una cuenta?
								</span>
							<Link
								to="/login"
								className="shadow-button hover-scale inline-flex items-center justify-center px-14 py-3.5 text-xl font-semibold leading-6 text-white whitespace-no-wrap bg-primary rounded-md"
								>
								Iniciar Sesión
							</Link>
							</>
						)}
					</div>
				</div>
			</section>
		</>
	);
};

export default SecondNav;
