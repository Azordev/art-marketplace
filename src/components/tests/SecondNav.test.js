import React from 'react';
import {MemoryRouter} from 'react-router';
import renderer from 'react-test-renderer';
import SecondNav from '../SecondNav';

it('renders correctly', () => {
  const tree = renderer
      .create(
          <MemoryRouter>
            <SecondNav />
          </MemoryRouter>,
      )
      .toJSON();
  expect(tree).toMatchSnapshot();
});
