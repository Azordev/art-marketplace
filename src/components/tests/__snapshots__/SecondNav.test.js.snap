// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="relative w-full sm:px-9 text-gray-700 bg-white body-font"
>
  <div
    className="max-w-screen-2xl container flex flex-col lg:flex-row justify-between py-5 px-0 mx-auto md:flex-row"
  >
    <div
      className="flex items-center justify-center md:justify-start"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="logo"
          className="block object-contain mx-w-17-4 mx-h-4-6 lg:w-17-4"
          src="logo.png"
        />
      </a>
    </div>
    <div
      className="relative flex-col lg:flex-row inline-flex items-center md:items-end lg:items-center lg:space-x-3"
    >
      <span
        className="inline-flex items-center lg:justify-center px-14 md:px-0 lg:px-14 py-3.5 text-xl font-semibold leading-6 text-black whitespace-no-wrap"
      >
        ¿Ya tienes una cuenta?
      </span>
      <a
        className="shadow-button hover-scale inline-flex items-center justify-center px-14 py-3.5 text-xl font-semibold leading-6 text-white whitespace-no-wrap bg-black rounded-md"
        href="/login"
        onClick={[Function]}
      >
        Iniciar Sesión
      </a>
    </div>
  </div>
</section>
`;
